daml-assistant/daml-helper/src/DA/Daml/Helper/Run.hs

```diff
@@ -728,9 +728,9 @@ runStart
                     , darPath
                     , "--script-name"
                     , initScript
-                    , if any (`elem` ["-w", "--wall-clock-time"]) sandboxOpts
-                        then "--wall-clock-time"
-                        else "--static-time"
+                    , if any (`elem` ["-s", "--static-time"]) sandboxOpts
+                        then "--static-time"
+                        else "--wall-clock-time"
```

Comment:
In my experience, mutually-exclusive switches either cause an error or operate on a "last one wins" principle. Is it worth tweaking this behavior so it follows one of these?

Reply:
Good point, I copied the way sandbox handles this since it seems to make sense to be consistent here. Also I really miss optparse-applicative now :slightly_smiling_face:

Reply:
I miss optparse-applicative in Scala too.